README-hi.md

```diff
@@ -0,0 +1,82 @@
+# कुबरनेट्स प्रलेखन
```

Comment:
This is probably me being paranoid.. :)

Hindi is written in the [Devanagri script](https://en.wikipedia.org/wiki/Devanagari)...could there be any problems with how this script is rendered with Hugo?

Reply:
I don't think there should be any problem with rendering, after all, it is just a text only.